app/serializers/version_serializer.rb

```diff
@@ -1,27 +1,47 @@
 class VersionSerializer < ActiveModel::Serializer
   embed :ids, include: true
   attributes :id,
-             :changes
+             :changes,
+             :whodunnit,
+             :previous_id,
+             :created_at
 
   has_one :discussion
+  has_one :comment
```

Comment:
Erg, we should really think about making this a polymorphic association. Not in this PR, but it's definitely technical debt we should pull out at some point.

```
# version.rb
belongs_to :versionable, polymorphic: true
```
```
#discussion.rb, comment.rb, proposal.rb
has_many :version, as: :versionable
```